cpp/ps0005/ps0005_solution.cpp

```diff
@@ -30,6 +24,7 @@ int main()
     cout << "Enter the operator + - * / %:" <<endl;
     cin >> op;

```

Comment:
- [x] declared variables to store the user input
- [x] chose suitable data types for the variables
- [x] prompted user to enter data
- [x] used `std::cin` in a right way to get user's input
+0.5 bonus mark for handling input process